### PR TITLE
Add deployment of Prometheus and Grafana to K8s

### PR DESCRIPTION
**Pull request by luoluoyuyu**
### Purpose
Add deployment of Prometheus and Grafana to K8s

### Remarks

PR introduces (a) breaking change(s): no

PR introduces (a) deprecation(s): no

### CODE REVIEW

**Comment by luoluoyuyu** [comment]
The results of the run are as follows
image

**Review by RobertIndie — APPROVED** [review]

File: installer/k8s/README.md [hunk 1]
```diff
@@ -168,7 +169,7 @@ rm -rf ${HOME}/streampipes-k8s
 | external.couchdb.appName                        | CouchDB application name                                 | "couchdb"                                |
 | external.couchdb.version                        | CouchDB version                                          | 3.3.1                                    |
 | external.couchdb.user                           | CouchDB admin username                                   | "admin"                                  |
-| external.couchdb.password                       | CouchDB admin password                                   | empty (auto-generated)                   |
+| external.couchdb.password                       | CouchDB admin password                                   | "admin"                                  |
```
Comment (dominikriemer) [inline installer/k8s/README.md:172]: Should we leave the CouchDB default as `auto-generated`?

Comment (luoluoyuyu) [reply installer/k8s/README.md]: already modified

**Comment by dominikriemer** [comment]
Awesome, thank you @luoluoyuyu!

**Pull request merged**